File: p001/p1.jl
#! /usr/bin/env julia
# Project Euler Problem #1

sum = 0

for i in 3:1000-1
	if i % 3 == 0 || i % 5 == 0
		sum += i
	end	
end	

println("The sum of all natural numbers <1000 and divisible by 3 or 5 is: $(sum)")

File: p001/p1.js
#! /usr/bin/env nodejs
// Project Euler Problem #1

var sums = 0;
for (var i = 1; i < 1000; i++){
	if (i % 3 == 0 || i % 5 == 0){
		sums += i;
	}
}

console.log("The sum of all natural numbers <1000 and divisible by 3 or 5 is:", sums)
